src/main/java/algorithms/orthogonalRangeSearching/oneDim/README.md

```diff
@@ -0,0 +1,54 @@
+# 1D Orthogonal Range Searching
+
+1D orthogonal range searching is a computational problem where you search for elements or data points within a 
+specified 1D range (interval) in a collection of 1D data (e.g. Find me everyone between ages 22 and 27). 
+
+While we can simply sort (and binary search the low and high of the specified range) to return all data points within 
```

Comment:
this is a wonderful addition. Maybe can slightly amend the phrasing to emphasize that preprocess by sorting is a valid solution - only if there are many more queries than update.

If data is updated / new insertions, then it will take O(n) to maintain sortedness for each update. (no need nlogn right? insertion can simply do O(n) pass to find insert point and another O(n) shift. Updating involves only bubbling that 1 element to its right place, the others will naturally fall into place)